MovingBlocks/Terasology
# Fixes various infos, warnings and errors from the checkstyle analysis

### Contains

Fixes for the huge amount of warnings from the checkstyle analysis in the build.

### How to test

Check checkstyle output with and without the pull request.
Change should not change the behaviour to the previous implementation.

### General

Maybe it would be a good idea to add a check to every pull request if it adds new violations. This would prevent adding more violations in the future.
